blog/_posts/2020-02-07-delete-clusters-faster-with-kind.md

```diff
@@ -0,0 +1,112 @@
+---
+slug: delete-clusters-faster-with-kind
+date: 2020-02-07
+author: nick
+layout: blog
+title: "Kill Your Kubernetes Clusters Faster with Kind"
+subtitle: "Announcing a Big Improvement in Local Development"
+tags:
+  - tilt
+keywords:
+  - tilt
+  - kind
+  - local development
+---
+
+At the ClusterAPI meeting in January
+
+Contributors create a Kind cluster and use Tilt for interactive development
+
+Slow, and hard to optimize. But fresh clusters were a huge improvement.
+
+That's why we're excited to announce some big changes to make Kind + Tilt faster than ever
+
+Use our setup script:
+
+TK LINK
+
+The script may look simple. 
+
+A lot of work from a lot of different people in the Kubernetes community went into making this possible.
+
+How does it work?
+
+## What is Kind?
+
+Kind runs a Kubernetes cluster in a Docker container 
```

Comment:
I think it's important to mention at the top that basically all advantages from Kind derive from the fact that it doesn't use a VM. And it might also be interesting to mention that drawbacks.

Reply:
what drawbacks did you have in mind?

Reply:
IIRC I read that if you're doing things that rely on the underlying mechanics of Kubernetes (e.g. writing CRDs), Kind doesn't mimic those as well as e.g. minikube? I might be wrong, but maybe it's prudent to google this for 5 minutes.

Reply:
i decided to just link to the pros/cons of different clusters at the bottom, which i think is better for the person who wants to know the trade-offs and drawbacks of Kind

Reply:
SGTM!